homeassistant/components/sensor/xbox_live.py

```diff
@@ -0,0 +1,94 @@
+"""
+Sensor for Xbox Live account status.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.xbox_live/
+"""
+import logging
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (CONF_API_KEY, STATE_UNKNOWN)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = 'mdi:xbox'
+
+REQUIREMENTS = ['xboxapi==0.1.1']
+
+CONF_XUID = 'xuid'
```

Comment:
That's the [official naming](https://xboxapi.com/documentation), stands for Xbox Account User ID. We could change it to `xbox_user_id`, although that seems a little long.